portfolio/src/main/webapp/about.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Olu's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div class="row">
+      <img src="images/logo.png"
+    </div>   
+    <ul class="main-nav">    
+      <li><a href="index.html" style="color:white"> HOME </a></li>
+      <li><a href="gallery.html" style="color:white"> PHOTOGRAPHY </a> </li>
+	  <li><a href="showcase.html" style="color:white"> PROJECT SHOWCASE </a></li>
+      <li class="active"><a href="about.html" style="color:white"> ABOUT </a></li>
+      <li><a href="contact.html" style="color:white"> CONTACT </a></li>
+      <li><a href="stock.html" style="color:white"> STOCK PICKER</a></li>
```

Comment:
Seems like these elements all repeat the same style. Consider using a CSS rule, instead of inline style.